common/controllers/form-wizard.js

```diff
@@ -17,14 +18,32 @@ class FormController extends Controller {
     next()
   }
 
+  setupAddMultipleFieldsValues(req, res, next) {
+    const values = req.sessionModel.toJSON()
+
+    const addAnotherProps = Object.keys(req.form.options.fields).filter(
```

Comment:
You could destructure/assign `req.form.options.fields` as you use it a few times in this method?

Reply:
No more or less than the other methods in the `form-wizard` module. ;)